README.md

```diff
@@ -365,6 +365,10 @@ chaplean_api_client:
 You can override the default email content by overriding the translation keys or even the email body twig template.
 The translation keys are under `chaplean_api_client.email.request_executed_notification` and the template is `Resources/views/Email/request_executed_notification.txt.twig`.
 
+To clean the logs from the Database, you may use the command `chaplean:api-logs:clean [minimumDate]`. It will remove all log entries past the minimum date. By default, this argument date is `now -1 month`. It should be formatted as a [PHP's DateTime string](https://www.php.net/manual/fr/datetime.formats.php).
```

Comment:
I think this should go into it's own section. Maybe a `Commands` section bellow `Additional Features`.

Also, the wording here is a bit weird. Either you remove entries older than a max date or you keep only entries more recent than min date but a mix of both wording sounds weird to me.

Maybe something along the lines of:
> It will remove old logs, only keeping those more recent than the given `minimumDate` date.